apps/src/blockly/utils.ts

```diff
@@ -16,19 +21,19 @@ export const TRUTHY_DEFAULT = attributeValue => attributeValue !== 'false';
  * @returns {boolean} The boolean value of the attribute as determined by the callback function.
  */
 export function readBooleanAttribute(
-  xmlElement,
-  attribute,
+  xmlElement: Element,
+  attribute: string,
   callback = FALSEY_DEFAULT
 ) {
   const attributeValue = xmlElement.getAttribute(attribute);
   return callback(attributeValue);
 }
 
-export function capitalizeFirstLetter(string) {
+export function capitalizeFirstLetter(string: string) {
```

Comment:
Should we be specifying return types?

Reply:
It's inferred here! Generally our rule of thumb (@sanchitmalhotra126  can correct me if he follows a different one) is if the type is inferred, no need to specify. It knows it's a string because it's a pretty simple string method.
<img width="528" alt="Screenshot 2024-02-01 at 9 35 02 AM" src="https://github.com/code-dot-org/code-dot-org/assets/33666587/1b98f8eb-0934-4e07-801a-35dfd07c999c">

Reply:
Agreed, I typically also leave out the return type if it's inferred. Sometimes it helps to be explicit if a function is more complex (for example it can be helpful to specify the return type of async functions like `Promise<SomeDataType>` so it doesn't default to `Promise<void>`) which also helps in enforcing that constraint throughout the function, but yeah in cases like this it seems fine to not specify.